.github/pull_request_template.md

```diff
@@ -1,26 +1,19 @@
-***Delete section if irrelevant***
+Fixes #(issue)
 
-### List related PRs if any
+## Type of change
 
-- pr link
+Please delete options that are not relevant.
 
-### List related issues if any
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Refactoring existing code
+- [ ] New Ruleset (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
I don't understand the purpose of the precision in parentheses here. Why couldn't a new ruleset just aim at increasing coverage? Maybe it's just the phrasing.

Reply:
Good catch, didn't realize I left the "default template" language that I borrowed from in here. I will remove that.